x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/job_type.tsx

```diff
@@ -22,23 +22,23 @@ export const JobType: FC<Props> = ({ type, setFormState }) => {
     'xpack.ml.dataframe.analytics.create.outlierDetectionHelpText',
     {
       defaultMessage:
-        'Outlier detection jobs require a source index that is mapped as a table-like data structure and analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
+        'Outlier detection identifies unusual data points in the dataset. Outlier detection jobs analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
     }
   );
 
   const regressionHelpText = i18n.translate(
     'xpack.ml.dataframe.analytics.create.outlierRegressionHelpText',
     {
       defaultMessage:
-        'Regression jobs analyze only numeric fields. Use the advanced editor to apply custom options, such as the prediction field name.',
+        'Regression makes predictions on your data after it determines certain relationships among your data points. Regression jobs analyze numeric, categorical, and boolean fields. Use the advanced editor to apply custom options.',
     }
   );
 
   const classificationHelpText = i18n.translate(
     'xpack.ml.dataframe.analytics.create.classificationHelpText',
     {
       defaultMessage:
-        'Classification jobs require a source index that is mapped as a table-like data structure and support fields that are numeric, boolean, text, keyword, or ip. Use the advanced editor to apply custom options, such as the prediction field name.',
+        'Classification predicts the class or category of a given data point in a dataset. Classification jobs support fields that are numeric, boolean, text, keyword, or ip. Use the advanced editor to apply custom options.',
```

Comment:
If the default message of outlier detection is limited to one sentence, then I suggest using only the first sentence here as well.

```suggestion
        'Classification predicts the class of a given data point in the data set. Classification jobs support fields that are numeric, boolean, text, keyword, or ip. Use the advanced editor to apply custom options.',
```

Reply:
Updated in f3feaac3a3bbf55688e365081e904507aef754bc

Reply:
>The UI uses the word `labels` in other places, so I suggest to use the same here. 

I spun up 7.9 and didn't see "labels" anywhere else in the wizard so if that concept is explained there it's not obvious. In the docs, we use phrases like this, which I prefer to "labels":

*  ...predicts the class or category of a data point in your data set (https://www.elastic.co/guide/en/machine-learning/master/dfa-classification.html)
* ...predict the value of the dependent variable (https://www.elastic.co/guide/en/machine-learning/master/flightdata-classification.html)
* ...predicts the value of a field in the document (the dependent variable) (https://www.elastic.co/guide/en/elasticsearch/reference/master/put-dfanalytics.html)

Alternatively, if we stick with "label", I think it needs to be clarified/defined somewhere in this same wizard.

Reply:
@lcawl Not in the wizard, but on the Analytics exploration page, we use "labels". I think we should either use the same term in other places or replace it with another solution. (Sorry for commenting on a merged PR, I didn't want to lose the context.)

<img width="1422" alt="Screenshot 2020-07-17 at 7 54 51" src="https://user-images.githubusercontent.com/22324794/87753563-276af700-c803-11ea-9b0a-84c65cdc76c2.png">